app/views/exports/hamilton_gem_x_5p_chip_loading.csv.erb

```diff
@@ -19,14 +19,16 @@
                       row_sep: ""
 %>
 <%
-  # Configuration specific to this export
+  # Constants from config/initializers/scrna_config.rb
+  scrna_config = Rails.application.config.scrna_config
 
-  required_number_of_cells = 30000 
-  wastage_factor = 0.95238 # accounting for wastage of material when transferring between labware
-  chip_loading_concentration = 2400 
-  desired_number_of_cells_per_chip_well = 90000
-  desired_sample_volume = 37.5 # microlitres
-  volume_taken_for_cell_counting = 10.0 # microlitres
+  required_number_of_cells_per_sample_in_pool = scrna_config[:required_number_of_cells_per_sample_in_pool]
+  wastage_factor = scrna_config[:wastage_factor]
+  desired_chip_loading_concentration = scrna_config[:desired_chip_loading_concentration]
+  desired_sample_volume = scrna_config[:desired_sample_volume]
+  volume_taken_for_cell_counting = scrna_config[:volume_taken_for_cell_counting]
+
+  number_of_cells_per_chip_well_key = scrna_config[:number_of_cells_per_chip_well_key]
```

Comment:
Not convinced all these assignments are worth it, as most of these variables are just used once 🤔

Reply:
not once, used for each source well in the plate, did it as wasn't sure of overhead of fetching multiple constants from the initializer file multiple times...? probably nothing
also makes the calculations hard to read if inline

Reply:
Ah I see, fair enough. Not sure. Happy as is.